index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare function sanitize(
+  input: string,
+  options?: { replacement?: string }
```

Comment:
`replacement` can also be a function ([ref](https://github.com/parshap/node-sanitize-filename/blob/master/README.md#sanitizeinputstring-options))